src/DebuggerData.h

```diff
@@ -1,10 +1,11 @@
 #ifndef DEBUGGERDATA_H
 #define DEBUGGERDATA_H
 
-#include <list>
 #include <QString>
 #include <QXmlStreamReader>
 #include <QXmlStreamWriter>
+#include <list>
```

Comment:
The #include for the `<list>` header doesn't seem to be needed (anymore?). Instead the #include for the `<vector>` header seems to be missing. (Was likely refactored in the past without adjusting the #includes).